rust/cardano-chain-follower/examples/follow_chains.rs

```diff
@@ -145,51 +137,49 @@ const RUNNING_UPDATE_INTERVAL: u64 = 100_000;
 /// Try and follow a chain continuously, from Genesis until Tip.
 #[allow(clippy::too_many_lines)]
 async fn follow_for(network: Network, matches: ArgMatches) {
-    info!(chain = network.to_string(), "Following");
-    let mut follower = ChainFollower::new(network, ORIGIN_POINT, TIP_POINT).await;
-
-    let all_tip_blocks = matches.get_flag("all-tip-blocks");
-    let all_live_blocks = matches.get_flag("all-live-blocks");
-    let stop_at_tip = matches.get_flag("stop-at-tip");
-    let halt_on_error = matches.get_flag("halt-on-error");
-    let log_bad_cip36 = matches.get_flag("log-bad-cip36");
-    let log_cip509 = matches.get_flag("log-cip509");
-    let log_raw_aux = matches.get_flag("log-raw-aux");
-    let largest_metadata = matches.get_flag("largest-metadata");
+    info!(network = network.to_string(), "Following");
+    let mut follower = ChainFollower::new(network, Point::ORIGIN, Point::TIP).await;
+
+    let is_all_tip_blocks = matches.get_flag("all-tip-blocks");
+    let is_all_live_blocks = matches.get_flag("all-live-blocks");
+    let is_stop_at_tip = matches.get_flag("stop-at-tip");
+    let is_halt_on_error = matches.get_flag("halt-on-error");
+    let is_log_raw_aux = matches.get_flag("log-raw-aux");
+
+    // Metadata
+    let is_log_bad_cip36 = matches.get_flag("log-bad-cip36");
```

Comment:
NAK, revert every single rename of a variable with `is_` in it.
Sorry, but I refuse to review this PR because of this.